coalib/results/Diff.py

```diff
@@ -20,11 +20,14 @@ def __init__(self, file_list=None,
         Creates an empty diff for the given file.
```

Comment:
commit message: Don't understand that:

> `Diff.add_lines` lines provided during instantiation of the Diff,
if provided.

what's provided now? :O